packages/connect-webextension/src/index.ts

```diff
@@ -216,6 +218,7 @@ const initProxyChannel = () => {
     channel.init();
     channel.on('message', message => {
         const { id, payload, type } = message;
+        if (!payload) return;
```

Comment:
Are we receiving events without payload and are they causing issues?

Reply:
I don't recall that specifically, probably it was just TS complaining